examples/rust_bindings/src/rust_bindings_formatted.rs

```diff
@@ -17,15 +23,16 @@ pub mod abigen_bindings {
             pub fn wallet(&self) -> WalletUnlocked {
                 self.wallet.clone()
             }
-            pub fn with_wallet(&self, mut wallet: WalletUnlocked) -> Result<Self, Error> {
+            pub fn with_wallet(&self, mut wallet: WalletUnlocked) -> Result<Self> {
                 let provider = self.wallet.get_provider()?;
                 wallet.set_provider(provider.clone());
                 Ok(Self {
                     contract_id: self.contract_id.clone(),
                     wallet,
+                    log_decoder: self.log_decoder.clone(),
                 })
             }
-            pub async fn get_balances(&self) -> Result<HashMap<String, u64>, Error> {
+            pub async fn get_balances(&self) -> Result<HashMap<String, u64>> {
```

Comment:
A quick reminder to change the title from `feat` to `feat!` and add the breaking label because this is a breaking change!

Reply:
It does look like a breaking change but this is just the example file of abigen expanded.

The actual breaking changes have long since been merged in master, it is only now that we're updating our docs to show how the generated code might look atm.